clustered/server/src/main/java/org/ehcache/clustered/server/management/ClusterTierManagement.java

```diff
@@ -102,37 +106,43 @@ public void init() {
     }
   }
 
-  public void clientConnected(ClientDescriptor clientDescriptor, ClusterTierClientState clientState) {
+  public void clientConnected(ClientDescriptor clientDescriptor) {
+    connectedClients.add(clientDescriptor);
     if (managementRegistry != null) {
       LOGGER.trace("clientConnected({})", clientDescriptor);
-      managementRegistry.registerAndRefresh(new ClusterTierClientStateBinding(clientDescriptor, clientState));
+      managementRegistry.registerAndRefresh(new ClusterTierClientStateBinding(clientDescriptor));
     }
   }
 
-
-  public void clientDisconnected(ClientDescriptor clientDescriptor, ClusterTierClientState clientState) {
+  public void clientDisconnected(ClientDescriptor clientDescriptor) {
+    connectedClients.remove(clientDescriptor);
     if (managementRegistry != null) {
       LOGGER.trace("clientDisconnected({})", clientDescriptor);
-      ClusterTierClientStateBinding clientStateBinding = new ClusterTierClientStateBinding(clientDescriptor, clientState);
+      ClusterTierClientStateBinding clientStateBinding = new ClusterTierClientStateBinding(clientDescriptor);
```

Comment:
you can rename the binding classes perhaps ?

Reply:
Yes. I know the name isn't accurate anymore. What do you want? `ClusterTierBinding`?

Reply:
ClientBinding ? DescriptorBinding ?